keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -61,24 +59,19 @@ class ApplicationController(
           summary
         }
       }
-      val constraintStates = if (hasDeliveryConfig) {
-        deliveryConfigRepository.constraintStateFor(application)
-      } else {
-        emptyList()
-      }
+      val constraintStates = deliveryConfigRepository.constraintStateFor(application)
 
       return mapOf(
         "applicationPaused" to resourcePauser.applicationIsPaused(application),
         "hasManagedResources" to resources.isNotEmpty(),
         "resources" to resources,
-        "hasDeliveryConfig" to hasDeliveryConfig,
```

Comment:
Everything always has a delivery config, so we don't need to include this field.